src/Uno.UI/UI/Xaml/Controls/ScrollViewer/ScrollViewer.SnapPoints.cs

```diff
@@ -366,6 +366,11 @@ private void AdjustOffsetsForSnapPoints(ref double? horizontalOffset, ref double
 		{
 			global::System.Diagnostics.Debug.Assert(staticZoomFactor == 1f || staticZoomFactor == zoomFactor);
 
+			if (snapPoints is null)
+			{
+				return null;
+			}
```

Comment:
There was a NullReferenceException happening here. It's likely not related to the issue though. Anyway, WinUI code has this null check.